cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunProfilerTask.java

```diff
@@ -52,6 +52,7 @@ public class RunProfilerTask implements Task {
     private final PrintStream err;
     private static final String JAVA_OPTS = "JAVA_OPTS";
     private static final String CURRENT_DIR_KEY = "current.dir";
+    private static final String JAVA_COMMAND = "java.command";
```

Comment:
There are other tasks like executable task, test task also use same system property. Shall move this to common util class for all tasks and refactor.

Reply:
Or we can remove the constant and use the string directly as other tasks to be consistant.